source/playbooks/playbook-permissions.rst

```diff
@@ -0,0 +1,85 @@
+
+Playbooks permissions
+---------------------
+
+The default Playbooks settings are completely open which enable all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions, and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, only Playbook Admins can edit playbooks. Team members can view public and private playbooks and runs, but can't edit playbooks, check off tasks, or remove members from runs. They also can't create public or private playbooks.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Scroll down to the **Playbook Admin** section and confirm that **Manage Public Playbooks** and **Manage Private Playbooks** are checked.
+4. Select **Save**.
+
+You can also set permissions for read-only playbooks that do allow members to create new public or private playbooks.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Then, check **Create Public Playbook** and **Create Private Playbook**.
+4. Select **Save**.
+
+I want to manage how playbooks are accessed
+-------------------------------------------
+
+A public playbook and its associated runs can be viewed by any member of a team. Playbooks can be converted from public to private, but once a playbook is private it can't be converted back to public.
+
+I want to restrict who can convert playbooks from public to private
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can control whether Members can convert playbooks from public to private.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, check **Convert Playbooks**.
+3. Select **Save**.
+
+To restrict this action so only Playbook Admins can convert playbooks from public to private:
+
+1. Go to **System Console > User Management > Permissions**.
+2. Scroll down to **Playbook Admin**.
+3. Check **Convert Playbooks**.
+4. Select **Save**.
+
+I want to control who starts a run
+----------------------------------
+
+By default, all Members can start a run using a playbook. You can restrict this so that only Playbook Admins can start a run. Note that with this configuration, Members are not able to start runs or edit playbooks.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Runs**. This also unchecks **Create Runs**.
+3. Scroll down to **Playbook Admin** and ensure that **Manage Runs** and **Create Runs** are checked.
```

Comment:
@crspeller, is the `Manage Runs` setting necessary here to limit who can run a playbook? Wondering if we can make the example simpler by focussing on just the one knob.

Reply:
They are nested and `Create Runs` is the only child so they are basically the same box. Clicking on one with check/uncheck the other.